tests/e2e/profile/profileData.js

```diff
@@ -0,0 +1,8 @@
+ const helpers = require('../helpers/helpers')
```

Comment:
I would rename this file to testData.js and the folder to just **data**

Reply:
Fixed